knife/Gemfile.lock

```diff
@@ -0,0 +1,347 @@
+GIT
+  remote: https://github.com/chef/chefstyle.git
```

Comment:
any particular reason to include this since it's not going into an omnibus build?

Reply:
Nope, I can remove it. Originally I had this set up to run stlye in the 'knife' dir, and not globally in the repo.